### DISCUSSION TITLE
每次打开设置时会打开雷蛇耳机的环绕音引擎程序

### DISCUSSION THREAD

**Discussion by wulongzhong** [shuax/MouseInc.Settings#141] [Q&A]
环绕音程序地址为
C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Razer\7.1 Surround Sound
C:\Program Files\Razer\RzAppEngine\rzappengine.exe

**Comment by shuax** [comment]
是打不开设置吗？还是额外开了个程序

**Comment by wulongzhong** [comment]
额外打开了这个程序

**Comment by shuax** [comment]
那估计是雷蛇程序的问题
